### PR TITLE
Readme.md: Add another workaround for the ASLR problems on Windows 10

### PR DESCRIPTION
**Pull request by t-b**

### CODE REVIEW

**Comment by t-b** [comment]
@timjarsky Ready.

File: Readme.md [hunk 1]
```diff
@@ -46,6 +46,15 @@ Steps:
   choose yes when asked for administrative permissions.
 - Restart Igor Pro
 
+You can check if that worked by looking in the Exploit-Protection (german:
+"Exploit-Schutz") submenu of Windows-Security (german: "Windows Sicherheit")
+under Programs. There should be entries for `C:\Program Files\WaveMetrics\...`.
+
+In case this did not work you can try the following approach:
+- Open a Powershell console as administrator
+- Go to the tools directory in this repo
```
Comment (timjarsky) [inline Readme.md]: @t-b let's give a bit more hand-holding here by including an example file path:

```.....\MIES\Packages\ITCXOP2\tools```

Comment (t-b) [reply Readme.md]: @timjarsky The path you are giving is a MIES path. The `tools` folder in this repo is at the top-level.

Comment (timjarsky) [reply Readme.md]: @t-b oops. So how about: .....\MIES\tools  ?  Some users may not be confident specifying the file path from the command line.

Comment (t-b) [reply Readme.md]: @timjarsky I've adapted the wording.

**Review by timjarsky — APPROVED** [review]

File: Readme.md (hunk 1 above)
Comment (timjarsky) [reply Readme.md]: @t-b thanks and now I understand what you were saying. Sorry I didn't catch on earlier.

**Pull request merged**